src/app/home/index/index.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="home-container">
```

Comment:
You have a home component, its not okay to have such prefix for each class inside. Remember that encapsulation we talked about ? The styles for home component will stay inside the home component, so you don't even need classes in the html. You can directly style by "<main>", "<section>", etc.